### ISSUE TITLE
Undefined constant  error Rector\Symfony\Set\SensiolabsSetList::FRAMEWORK_EXTRA_61 

### ISSUE THREAD

**Issue by 77web** [quartetcom/static-analysis-kit#139]
composer run analyse fails with "undefined constact Rector\Symfony\Set\SensiolabsSetList::FRAMEWORK_EXTRA_61 " although rector/rector-symfony exists in vendor/rector/rector/vendor/rector/rector-symfony.
I'm using this kit in my standalone library, not in a Symfony full-stack application. But contains some of symfony components.

- php 8.2.10
- quartetcom/static-analysis-kit v8.2.4
- rector/rector 0.17.13

## error output

```
% composer run analyse
> @php static-analysis-kit analyse

Running rector
==============
                                                                                
 [ERROR] Undefined constant                                                     
         Rector\Symfony\Set\SensiolabsSetList::FRAMEWORK_EXTRA_61               
                                                                                

Script @php static-analysis-kit analyse handling the analyse event returned with error code 1
```

**Comment by 77web** [comment]
reproduced on CI:
https://github.com/77web/sendgrid-marketing-campaign-api-php-client/actions/runs/6839602776/job/18597955695

**Comment by siketyan** [comment]
@77web Thank you for reporting this. Actually this is an known issue and (temporarily) can be fixed via pinning `rector/rector` to `v0.17.1`. Will going to upgrade v0.18 though. Sorry for inconvenience.

**Comment by siketyan** [comment]
Fix released in v8.1.18 and v8.2.5.
